hotel-management/src/types/index.ts

```diff
@@ -6,3 +6,9 @@ export type TextProps = {
 export type LinkProps = TextProps & {
   href: string;
 };
```

Comment:
What happens if there are more types in the future, are those will belong to this file too?
Please split these types based on entities